copier/cli.py

```diff
@@ -25,6 +33,22 @@ def _wrapper(*args, **kwargs):
 
 
 class CopierApp(cli.Application):
+    """
+    The Plumbum CLI application.
```

Comment:
I tried always to follow [PEP 257](https://www.python.org/dev/peps/pep-0257/) which says:

>  The summary line may be on the same line as the opening quotes or on the next line. The entire docstring is indented the same as the quotes at its first line (see example below).

So, this should be:

```suggestion
    """The Plumbum CLI application.
```

I noticed it doesn't alter the mkdocs result, so please change this.

Reply:
Based on the PEP description, I follow it as well :stuck_out_tongue: 

But of course, let's keep the existing style! I was sleepy when I committed last night ^^